packages/react-sdk/src/css/css.ts

```diff
@@ -47,10 +65,14 @@ export const generateCssText = (data: Data) => {
 
   const styleRules = getStyleRules(styles, styleSourceSelections);
   for (const { breakpointId, instanceId, style } of styleRules) {
-    engine.addStyleRule(`[${idAttribute}="${instanceId}"]`, {
-      breakpoint: breakpointId,
-      style,
-    });
+    engine.addStyleRule(
+      `[${idAttribute}="${instanceId}"]`,
+      {
+        breakpoint: breakpointId,
+        style,
+      },
+      createImageValueTransformer(assets)
```

Comment:
do you want this transformer passed to the rule specifically or to the engine? it seems to me it could be on createCssEngine()

Reply:
Transformer accepts recent "assets". In some cases from useStore(assetsStore), in other with assetsStore.get(). Not sure how we can provide it globally.